ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -326,38 +332,40 @@
     },
     {
       'locale': 'fr',
-      'missingTranslations': true,
       'replaceSync': {
-        'Bladesblood': 'onde de choc',
+        'Bladesblood': 'Onde De Choc',
         'Gunshield': 'bouclier-canon',
         'I shall not yield!': 'Mon pavois est infrangible!',
         'Ignis Est': 'Ignis Est',
         'Magitek Turret II': 'tourelle magitek TM-II',
-        'Phantom Varis': 'double de Varis',
+        'Phantom Varis': 'Double De Varis',
         'Terminus Est': 'Terminus Est',
         'Varis Yae Galvus': 'Varis yae Galvus',
         'Ventus Est': 'Ventus Est',
       },
       'replaceText': {
-        'Aetherochemical Grenado': 'Grenade magitek',
-        'Alea Iacta Est': 'Alea Jacta Est',
-        'Altius': 'Altius',
-        'Citius': 'Citius',
-        'Electrified Gunshield': 'Bouclier-canon : Choc magitek',
-        'Festina Lente': 'Festina Lente',
+        '--clones appear--': '--Apparition des clones--',
+        '--clones appear?--': '--Apparition des clones ?--',
+        'Aetherochemical Grenado?': 'Grenade Magitek ?',
+        'Alea Iacta Est?': 'Alea Jacta Est ?',
+        'Altius?': 'Altius ?',
+        'Blade\'s Pulse': '
```

Comment:
Missing a translation here?

Reply:
i search, no ID

Reply:
Yeah, I don't see it either in Saint Coinach, sorry.  It's the thing that appears that tanks need to click on in order to block the big attack in phase 2.  Like the one in Susano.

Can you remove this for now if you don't know what it is?

Reply:
yep